lib/src/models/data/sports.dart

```diff
@@ -1,9 +1,8 @@
 import "dart:async";
 
-import "package:ramaz/constants.dart" show DayComparison;
+import "package:ramaz/constants.dart";
 import "package:ramaz/data.dart";
 import "package:ramaz/services.dart";
-
 import "model.dart";
```

Comment:
I like to keep a line splitting the different types of imports. Broadly, you have, in the following order:

```dart
// dart imports
import "dart:async";
import "dart:io";

// flutter imports
import "package:flutter/material.dart";
import "package:flutter/foundation.dart";

// third-party imports
import "package:cloud_firestore/cloud_firestore.dart";
import "package:firebase_auth/firebase_auth.dart";

// ramlife imports 
import "package:ramaz/models.dart";
import "package:ramaz/services.dart";

// relative imports (in the same folder)
import "model.dart";
import "schedule.dart";
```